sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/MergeRows.scala

```diff
@@ -87,7 +87,11 @@ object MergeRows {
     override def dataType: DataType = NullType
   }
 
-  case class Keep(condition: Expression, output: Seq[Expression]) extends Instruction {
+  case class Keep(
+      condition: Expression,
+      output: Seq[Expression],
+      isSystem: Boolean = false)
```

Comment:
can we add a small comment to explain what this flag means?

Reply:
done